modules/virt-creating-vm-snapshot-cli.adoc

```diff
@@ -42,6 +49,27 @@ of the `VirtualMachineSnapshot` object.
 $ oc create -f <my-vmsnapshot>.yaml
 ----
 
+. Optionally, if you are taking an online snapshot, you can use the `wait` command and monitor the status of the snapshot.
+.. Enter the following command:
++
+[source,terminal]
+----
+$ kubectl wait my-vm my-vmsnapshot --for condition=Ready
+----
+.. Verify the status of the snapshot.
+* `InProgress` - The online snapshot operation is still in progress.
+* `Succeeded` - The online snapshot operation completed successfully.
+* `Failed` - The online snapshot operaton failed.
++
+[NOTE]
```

Comment:
Good question. The only reason I called it out is because this seemed like a common failure and we weren't headlining this in its own module. I'm tempted to keep it as a note.